helm/managed/observability/eck/otlp-http-collector-k8s.yaml

```diff
@@ -4,8 +4,3 @@ global:
     - type: OTLP-HTTP
       endpoint: "http://eck-apm-opentelemetry-collector.default.svc.cluster.local:4318/v1/traces"
       consoleDebug: true
-      batch:
```

Comment:
You may have already mentioned this, but why remove the batching?

Reply:
It's not relevant to this sample configuration.